CHANGES.md

```diff
@@ -1,5 +1,10 @@
 ## Changes in 0.6.2 (in development)
 
+* The `xcube gen2` data cube generator tool has been hidden from
```

Comment:
When I have this branch checked out and call `xcube --help` I would expect `gen2` not to appear in the list because it is supposed to be hidden. I have done `python setup.py develop` beforehand. 

![image](https://user-images.githubusercontent.com/23457217/108474316-7e501780-728f-11eb-8896-db5361983fcd.png)

Reply:
It _does_ not appear. It is a problem of your env.